src/components/DataSubmissions/DataUpload.tsx

```diff
@@ -1,386 +1,142 @@
-import { ReactElement, useEffect, useMemo, useRef, useState } from "react";
-import { useMutation } from "@apollo/client";
-import { useParams } from "react-router-dom";
-import { LoadingButton } from "@mui/lab";
-import { VariantType } from "notistack";
-import { Button, Stack, Typography, styled } from "@mui/material";
-import RadioInput from "./RadioInput";
-import { CREATE_BATCH, CreateBatchResp, UPDATE_BATCH, UpdateBatchResp } from "../../graphql";
-import { useAuthContext } from "../Contexts/AuthContext";
-import DeleteDialog from "../../content/dataSubmissions/DeleteDialog";
+import { FC, ReactElement, useMemo, useState } from "react";
+import { useLazyQuery } from "@apollo/client";
+import { useSnackbar } from "notistack";
+import { Box, Button, Typography, styled } from "@mui/material";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import env from "../../env";
+import { RETRIEVE_CLI_CONFIG, RetrieveCLIConfigResp } from "../../graphql";
+import { downloadBlob, filterAlphaNumeric } from "../../utils";
 import FlowWrapper from "./FlowWrapper";
+import UploaderToolDialog from "../UploaderToolDialog";
+import UploaderConfigDialog, { InputForm } from "../UploaderConfigDialog";
 
-const StyledUploadTypeText = styled(Typography)(() => ({
-  color: "#083A50",
-  fontFamily: "'Nunito', 'Rubik', sans-serif",
-  fontSize: "16px",
-  fontStyle: "normal",
-  fontWeight: 700,
-  lineHeight: "19.6px",
-}));
+export type Props = {
+  /**
+   * The submission to download a pre-configured CLI config for.
+   */
+  submission: Submission;
+};
 
-const StyledMetadataText = styled(StyledUploadTypeText)(() => ({
-  "&.MuiTypography-root": {
-    color: "#000000",
-  },
-}));
+const StyledBox = styled(Box)({
+  maxWidth: "790px",
+  lineHeight: "22px",
+  marginTop: "10px",
+});
 
-const StyledUploadFilesButton = styled(Button)(() => ({
-  minWidth: "137px",
-  padding: "10px",
-  color: "#FFF",
-  fontFamily: "'Nunito'",
-  fontSize: "16px",
+const StyledDownloadButton = styled(Button)({
+  padding: "10px 6px",
+  fontSize: "14px",
   fontStyle: "normal",
-  lineHeight: "24px",
+  lineHeight: "16px",
   letterSpacing: "0.32px",
-  textTransform: "none",
+  width: "137px",
+  height: "44px",
```

Comment:
Other buttons are 46px, we should keep this consistent.

Reply:
Good catch. It seems the buttons have a calculated height of 47px, so I updated the styling to that.  a6b7266

For this button, we can't omit the height attribute, otherwise it grows too big (multiple text lines).